addons/kotlin/plugin.cfg

```diff
@@ -0,0 +1,7 @@
+[plugin]
+
+name="Godot Kotlin"
+description="Adds support for writing Godot scripts in Kotlin via GDNative"
+author="Adam Brown"
```

Comment:
Your Name here is good as you are the author but if you don't mind i would write something like this:
`Adam Brown and Godot-Kotlin Contributors` or `Adam Brown for Godot-Kotlin` or `Adam Brown for Utopia-Rise` so people in the asset store see it's the official plugin.

Also maybe add the Link to the "real" repo in the description and maybe add a line like:
"Official Plugin for Godot-Kotlin by Utopia-Rise and Contributors" or something like that.
I think that gives the plugin more credibility in the asset store